apps/SageAccountWeb/src/components/AccountSettings.tsx

```diff
@@ -572,8 +585,30 @@ export const AccountSettings = () => {
                 </div>
                 <div className="credential-partition">
                   <h4>Profile Validation</h4>
-                  <CompletionStatus isComplete={verified} />
-                  <p>
+                  {(verificationState === undefined ||
+                    verificationState.state ===
+                      VerificationStateEnum.APPROVED) && (
+                    <CompletionStatus isComplete={verified} />
+                  )}
```

Comment:
Is this appropriate to show if `verificationState === undefined`? I would expect a skeleton (or nothing)

Reply:
This is the case before a submission has been created, so verified is false (and this UI would show Incomplete in red).

Reply:
I completely missed the `isComplete` prop (which didn't even change). Totally makes sense!